### DISCUSSION TITLE
Something and Other — according to Houlgate

### DISCUSSION THREAD

**Discussion by ghost** [systemphil/sphil#97] [Hegel]
Hello

Reading Houlgate's chapter 8 "Something and Other" of his book "On Being", I was stricken by what seems to be a discrepancy between Hegel's categories in Science of Logic and in the Encyclopedia. I know there can be differences but they seem more pronounced in the section on Essence. Here I am wondering whether Houlgate's interpretation may induce this discrepancy.

In the Encyclopedia, it seems that the Other is first introduced as "Anderssein", that is, as an intimate distorsion in the Something. In other words, this Other is "immanent" to the Something, this is what makes it change.

But Houlgate seems to introduce directly the Other as "an other something", which is a bigger claim. He does quote Hegel writing that "something and the other are _somethings_ (SL 90 / LS 112)". But since Houlgate admits he is himself reconstructing an argument that was lacking in Hegel (who uses an anticipatory remark instead of an immanent logic) there _may_ be a misunderstanding here. 

Indeed, Houlgate says that Negation has a self-relation, and thus is also something (like Reality was). BUT, is not Houlgate forgetting that Negation and Reality are already sublated in the concept of Something ? Of course, deriving the self-relation of Negation is possible insofar as Negation is preserved, but its suppression or "subjugation" should not be forgotten. This subjugation would entail that "an other something" is to be taken only _relatively_, or "internally" to Something, and not as an other something that stands apart. This kind of otherness would come a bit later.

More than settling the problem, I hope that this discrepancy makes sense for the reader.

**Comment by Firgrep** [comment]
I'm unsure about the idea of "self-relating" negation and what role it plays here. Relations tend to be more prominent in essence, whereas in being the transitions are immediate. Perhaps you can elaborate what negation does here?

I'll be looking closely at `something` and `other` as I start work on `existence`, but if memory serves, `other` both is internal to `something` and something independent, since what it means to be `something` is that it entails some interior (and hence otherness proper).  What do you think?

**Comment by ghost** [comment]
I have followed compliantly Houlgate’s reading, and I will tell you how I understand this self-relating : after we get Dasein, which is being with an immediate determination, we obtain Reality by saying : « hey, but Dasein has in fact both being and non-being »… so Reality is the same as Dasein, except that its non-being is less immediate. And since the non-being has now been flushed out, we can also see Dasein the other way around : this is a non-being that has being, and that is the category of Negation. So Reality and Negation are really the two sides of the same coin. They are different and yet the same : Reality is Negation and yet different from it — and vice-versa. This is what, at this point, Houlgate calls a self-relation, following Hegel : « simple self-relation in the form of being » (einfache, seiende Beziehung auf sich).

Yes I agree that to be something entails some interior, but we may distinguish between a first level which is the simple self-relating in question, called « being within itself » (Insichsein) and the later category of « being in itself » (Ansichsein) which will stand in direct contrast to an external other. So what I mean is that the first kind of interiority is related only to an internal form of otherness, whereas it seems to me that Houlgate introduces the external form of otherness (otherness proper) already at this point.

Hope it is clear.

**Comment by Firgrep** [comment]
Yes, that's helpful. Although I think the relation in question is a sort of shorthand for the logic you just described, which really boils down to a series of immediate transitions. I'm not sure there exists true relations in the Doctrine of Being, as in Essence, but your comment got me thinking and I'll be attentive to it when I look closer at those sections. 

The points you make about interiority/exteriority are interesting. Once again, I cannot add very much without having the text in mind. Would you consider reviewing the articles on existence and something when we have drafts ready?

**Comment by ghost** [comment]
You are probably right regarding the notion of « relation » ; but I don’t know this matter well enough. Yes of course I can have a look at your drafts !
